vault/resource_kubernetes_secret_backend_role.go

```diff
@@ -143,6 +155,7 @@ func kubernetesSecretBackendRoleCreateUpdate(ctx context.Context, d *schema.Reso
 	data := make(map[string]interface{})
 	fields := []string{
 		fieldAllowedKubernetesNamespaces,
+		fieldAllowedKubernetesNamespaceSelector,
```

Comment:
Looks like allowed_kubernetes_namespace_selector was added in Vault 1.12. We should add a Vault Server version check in the resource and test files. You can see a examples of doing version checks in this PR: https://github.com/hashicorp/terraform-provider-vault/pull/1936/files#diff-3c100f49c020a28974d882dd5b34d276a7f825e3ccfaa18e584c1e26e2261d10R237

Reply:
Fixed